"TypeError: undefined is not a function" in typescript lib.

I get this error on mac os 10.10.5, npm 2.14.2, node v0.12.7.

black-screen version 7b11841a50c4dcde25ad4c9c8b24ad918e2b7878.

```sh
$ npm run install-all
...
# No errors.

$ gulp
[17:35:09] Using gulpfile ~/black-screen/gulpfile.js
[17:35:09] Starting 'default'...
[17:35:09] Starting 'watch'...
[17:35:09] Starting 'clean'...
[17:35:09] Finished 'default' after 359 ms
[17:35:09] Finished 'clean' after 58 ms
[17:35:09] Starting 'typescript'...
[17:35:09] Starting 'sass'...
[17:35:10] Starting 'react'...
[17:35:11] gulp-notify: [Black Screen Watcher] React has been compiled.
[17:35:11] Finished 'react' after 1.83 s
/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35171
            if (host.fileExists(fileName)) {
                     ^
TypeError: undefined is not a function
    at tryLoad (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35171:22)
    at loadNodeModuleFromFile (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35164:20)
    at loadModuleFromNodeModules (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35213:30)
    at nodeModuleNameResolver (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35158:20)
    at resolveModuleName (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35140:41)
    at /Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35417:113
    at Object.map (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:945:29)
    at resolveModuleNamesWorker (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35417:65)
    at processImportedModules (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35833:35)
    at findSourceFile (/Users/agkish/black-screen/node_modules/typescript/lib/typescript.js:35795:21)
```

+1 I get the same error:
```
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.10.5
BuildVersion:	14F27

$gulp
...
[4667:0906/001730:INFO:CONSOLE(336)] "Uncaught Error: Cannot find module '/Users/me/dev/black-screen/compiled/src/views/TerminalLayout'", source: module.js (336)

```

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "gulp-react": "3.0.1",
     "gulp-sass": "2.0.4",
     "gulp-shell": "0.4.2",
-    "gulp-typescript": "2.8.0",
+    "gulp-typescript": "2.8.2",
     "gulp-util": "3.0.6",
     "mocha": "2.2.5",
     "react-tools": "0.13.3",

